vagga.yaml

```diff
@@ -15,6 +15,30 @@ commands:
     container: ubuntu
     run: [cargo, test]
 
+  gen-ubuntu-idx: !Command
```

Comment:
Some indication that this is just a test data would be fine, maybe `gen-test-data`? Similarly in description we should indicate "Generate ... image, used for benchmarking algorithms`